PHP/CalculateMerchantApiHash.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+/**
+ * This is example for calculating Merchant API authentication hash in case of refund.
+ * Merchant API GET requests have always empty content, in this case use empty string as $requestContent variable when calculating content MD5
+ */
+
+$paymentId = 102402728626; // Payment id is returned after returning from payment back, $_GET['PAYMENT_ID'];
+
+// Test merchant credentials
+$merchantId = 13466;
+$merchantSecret = '6pKF4jkv97zmqBJ3ZL8gUw5DfT2NMQ';
+
+$url = 'https://api.paytrail.com/merchant/v1/payments/' . $paymentId . '/refunds';
+$method = 'POST'; // Creating refund uses POST method
+
+// Product we want to refund
+$product = [
+   'amount' => 1000, // Price is in cents, in this case price is 10.00€
+   'description' => 'Test Product',
+   'vatPercent' => 2400, // Vat 24.00%, comma is removed
+];
+
+// Create message content
+$content = [
+   'rows' => [
+      $product,
+   ],
+   'email' => 'customer@email.com', // Customer email address
+   'notifyUrl' => 'https://url.to.shop/apiNotification/', // Paytrail API will send return message to this url
+];
+
+$requestContent = json_encode($content);
+/* Output is following
+{"rows":[{"amount":1000,"description":"Test Product","vatPercent":2400}],"email":"customer@email.com","notifyUrl":"https:\/\/url.to.shop\/apiNotification\/"}
+*/
+
+$method = 'POST'; // For creating refund, method is POST

```

Comment:
Already declared on line 15.

Reply:
Removed obsolete assign and changed timestamp assign to use correct function.